internal/runbits/requirements/rationalize.go

```diff
@@ -71,6 +77,32 @@ func (r *RequirementOperation) rationalizeError(err *error) {
 			errs.SetInput(),
 		)
 
+	case errors.As(*err, &multipleMatchesErr):
+		messages := []string{
+			locale.Tl("err_package_multiple_matches",
+				"There are multiple matches for '[ACTIONABLE]{{.V0}}[/RESET]'. Please specify the requirement you want to install by typing its full namespace.",
+				multipleMatchesErr.Query),
+			"", // blank line
+			locale.Tl("err_package_multiple_matches_found", "Matches found:"),
+			"", // blank line
+			locale.Tl("err_package_multiple_matches_header", "  [HEADING]Name\t\t\tType\t\t\tNamespace[/RESET]"),
+			"", // blank line
+		}
+		for _, match := range multipleMatchesErr.Matches {
+			namespace := model.NamespaceFromIngredient(match)
+			typeString := namespace.Type().String()
+			if lang := model.LanguageFromNamespace(namespace.String()); lang != "" {
+				typeString = fmt.Sprintf("%s %s", cases.Title(language.English).String(lang), typeString)
+			}
+			messages = append(messages,
+				locale.Tl("err_package_multiple_matches_match",
+					"  {{.V0}}\t\t\t{{.V1}}\t\t[ACTIONABLE]{{.V2}}[/RESET]",
+					*match.Ingredient.Name,
+					typeString,
+					fmt.Sprintf("%s/%s", *match.Ingredient.PrimaryNamespace, *match.Ingredient.Name)))
+		}
+		*err = errs.WrapUserFacing(*err, strings.Join(messages, "\n"), errs.SetInput())
```

Comment:
Yes, the use of tab characters for formatting is awful, but our errors and user-facing errors mechanism expects strings, not objects, so I cannot pass the usual struct with tags to print nicely as a table. I'm open to suggestions or a redo of the mockup.